lib/mongoose_push_web/controllers/dummy_controller.ex

```diff
@@ -0,0 +1,7 @@
+defmodule MongoosePushWeb.DummyController do
```

Comment:
This file uses 4 spaces indentation, while other use double. We should normalize to double-spaces.